### PR TITLE
Componente input creado y funcionando correctamente

### PR DESCRIPTION
**Pull request by cristinapicatoste**

### CODE REVIEW

File: src/pages/Register/Register.css [hunk 1]
```diff
@@ -1,25 +1,40 @@
-/* body {
```
Comment (PabloIraburu) [inline src/pages/Register/Register.css:1]: I understand that the background and the fonts are not used anymore?

File: src/pages/Register/Register.js [hunk 2]
```diff
@@ -39,9 +40,12 @@ export const Register = () => {
         <div className="Register">
             <h1>Súmate a la mejor plataforma de música</h1>
             <p>Accede a millones de canciones, de todos los estilos, en todos tus dispositivos. Solo te llevará 30 segundos.</p>
-            <input type="text" name="name" onChange={handleInput} ref={register({ required: true, maxLength: 8 })} placeholder="Name" />
-            <input type="email" name="email" onChange={handleInput} ref={register({ required: true })} placeholder="Email" />
-            <input type="password" name="password" onChange={handleInput} ref={register({ pattern: /^[A-Za-z0-9]+$/i })} placeholder="Password" />
+            <Input type={"text"} name={"name"} onChange={handleInput} placeholder={"Name"}/>
+            <Input type={"email"} name={"email"} onChange={handleInput} placeholder={"E-mail"}/>
+            <Input type={"password"} name={"password"} onChange={handleInput} placeholder={"Password"}/>
+            {/* <input type="text" name="name" onChange={handleInput} ref={register({ required: true, maxLength: 8 })} placeholder="Name" /> */}
```
Comment (PabloIraburu) [inline src/pages/Register/Register.js:46]: If the old input component is being replaced by yours feel free to erase old calls, no need to comment them

**Review by PabloIraburu — COMMENTED** [review]
Everything looks good, the only question I have is about the register.css background color but I'm sure you had it into account!

File: src/pages/Register/Register.css (hunk 1 above)
Comment (cristinapicatoste) [reply src/pages/Register/Register.css:1]: We will set the fonts for the whole app at app.css (or similar) to not repeat it in all components

**Pull request merged**